package.json

```diff
@@ -8,9 +8,9 @@
     "url": "https://github.com/hawkular/hawkinit"
   },
   "scripts": {
-    "test": "echo \"WARN: no test specified; todo: add some tests :]\" && exit 0",
+    "test": "mocha specs/*",
```

Comment:
do we have to add it in the travis.yml? I assume, the tests are being run during the install and we do the install. Correct me if I am wrong?